### PR TITLE
Boundary documentation: updated domain model page

### PR DESCRIPTION
**Pull request by taoism4504**
Per Pete's request, the following **domain model** overview page was updated to reflect new content found [here](https://docs.google.com/document/d/14_ebLDnVO8Cw7dHNZ9Ahh34_LK0HjQUKlYaou8VwT60/edit#).

:mag: [Deploy Preview](https://boundary-git-update-domain-model-hashicorp.vercel.app/docs/concepts/domain-model)

### CODE REVIEW

File: website/content/docs/concepts/domain-model/index.mdx [hunk 1]
```diff
@@ -7,94 +7,129 @@ description: |-
...
+
 
 ![](/img/boundary-domain-model-overview.jpg)
 
-## Resources
+The highest-level  scope is the [Global Scope](/docs/concepts/domain-model/scopes#global). Boundary administrators operate on this level to configure and manage Boundary for the entire business. Boundary administrators  can also manage child scopes, auth methods, users, groups, roles, and grants.
+
+The intermediate scope level is  [Organizations](/docs/concepts/domain-model/scopes#organizations), also referred to as orgs, which are a child scope of global.  Though the implementation may be flexible for organization scopes, we recommend scoping by business units in this case. Scoping by business unit provides a clear separation of concerns for each business unit. Creating organizations for the development, operations, and support teams would be an example.
+
+The lowest scope level is the [Project](/docs/concepts/domain-model/scopes#projects). An organization’s scope may have as many projects as required based on its business workflows. To expand on the example used earlier, the development team’s organization can also have project scopes for their different products.
```
Comment (stellarsquall) [inline website/content/docs/concepts/domain-model/index.mdx:24]: I'm ignorant here, what's meant by "lowest?" Lowest in the global scope?

File: website/content/docs/concepts/domain-model/index.mdx [hunk 2]
```diff
@@ -7,94 +7,129 @@ description: |-
...
+
+Users are associated with either the global or an organization’s scope. For better access control management, users may be positioned into groups to ease access  control demands. Groups can also contain users from different organizations or global scopes without creating multiple users for a single entity, making permissioning less complicated to implement.
+
+An [Auth Method](/docs/concepts/domain-model/auth-methods) is the mechanism in Boundary used for authentication. An organization may use numerous authentication methods. Boundary supports Open ID Connect (OIDC) auth methods to delegate authentication to an external identity provider (IDP) - such as Azure Active Directory, Okta, and other OIDC providers - as well as a Boundary-native password auth method. Within an auth method, [Accounts](/docs/concepts/domain-model/accounts) may be created and associated with users.  Once associated  with a user, the principal may authenticate in Boundary using the method specified by their auth method. An account for the password auth method consists of a login name and password.
```
Comment (xingluw) [inline website/content/docs/concepts/domain-model/index.mdx:38]: Could be useful to provide an example of a user having multiple accounts. For example, being connected with OIDC as one but also having a username-password in break-glass scenarios.

File: website/content/docs/concepts/domain-model/index.mdx [hunk 3]
```diff
@@ -7,94 +7,129 @@ description: |-
...
+The diagram below illustrates the relationship between the different IAM components within Boundary.
+
+![](/img/component-relationship.png)
+
+
+## Target and Host Resources
+
+Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single  session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to one of its host sources with returned credentials from the target’s credential source.
+
+![](/img/target-host-resources.png)
```
Comment (xingluw) [inline website/content/docs/concepts/domain-model/index.mdx:58]: The quality of this image looks a lot worse than the same one above?

**Review by xingluw — APPROVED** [review]
LGTM

File: website/content/docs/concepts/domain-model/index.mdx (hunk 3 above)
Comment (stellarsquall) [reply website/content/docs/concepts/domain-model/index.mdx:58]: it's true that both component-relationship.png and target-host-resources.png seem to be much smaller than the resources-overview.png at the top

File: website/content/docs/concepts/domain-model/index.mdx (hunk 2 above)
Comment (taoism4504) [reply website/content/docs/concepts/domain-model/index.mdx:38]: @PPacent , can you help fill in the content gaps here per xingluw's suggestions? Thanks.

File: website/content/docs/concepts/domain-model/index.mdx (hunk 3 above)
Comment (taoism4504) [reply website/content/docs/concepts/domain-model/index.mdx:58]: I will see about re-adjusting the images to improve the quality.

File: website/content/docs/concepts/domain-model/index.mdx (hunk 2 above)
Comment (PPacent) [reply website/content/docs/concepts/domain-model/index.mdx:38]: @taoism4504 you can use the following:

"A user may be associated with accounts from different auth methods, such as a user having both an Azure Active Directory account and an Okta account. In this case, the user could authenticate with either provider."

**Review by stellarsquall — APPROVED** [review]
great work!

**Pull request merged**